#105.A
def call_exceptor():
    print("call_exceptor starts here...")
    no_return()
    print("an exception is raised")
    print("....so this won't print")
